### DISCUSSION TITLE
Album or tag categorization based on a fenced location

### DISCUSSION THREAD

**Discussion by Nikishek** [immich-app/immich#12189] [General]
I think no editor has done that so far. :-)

- This could work automatically based on some predefined conditions (and be part of some larger autosorting feature)
- or as a one-time sorting command.

For example, you're moving around town. Every time you take photos in the park, they will be assigned to the _In the Park_ album. Or under the same tag.

**Comment by radh21301** [comment]
This feature request covers what you need and more.
https://github.com/immich-app/immich/discussions/1673

Can you close this request if that looks like what you want?
